operators/monitor-tcp-probe.yml

```diff
@@ -0,0 +1,48 @@
+# Exporter jobs
+- type: replace
+  path: /instance_groups/name=prometheus2/jobs/-
+  value:
+    name: blackbox_exporter
```

Comment:
I dont think this will work - the blackbox_exporter job is already added in monitor-http-probe.yml and I think you cant add the same job twice...

Reply:
@gordcorp Good point, I was hoping to avoid this but I'll assume bosh can only define and run a job once :disappointed: 

I've merged the module and task into the `blackbox_exporter` job.